backend/chromeApiServer/src/main/java/com/canvas/service/helperServices/CanvasClientService.java

```diff
@@ -130,6 +129,9 @@ public byte[] fetchFileUnderCourseAssignmentFolder(ExtensionUser user, String fi
             JsonNode filesResponse = parseResponseToJsonNode(this.okHttpClient.newCall(filesRequest).execute());
             String fileId = getFileIdFromFilesResponse(filesResponse, fileName);
             return fetchFile(fileId, user.getBearerToken());
+        } catch (MakefileNotFoundException makefileNotFoundException) {
+            logger.error(makefileNotFoundException.getMessage());
+            throw makefileNotFoundException;
         } catch (Exception e) {
             throw throwCanvasException(e);
```

Comment:
Should we log this exception too?

Reply:
it's already being logged in the throwCanvasException method